CMakeLists.txt

```diff
@@ -3144,6 +3144,14 @@ if(SDL_STATIC)
   else()
     set_target_properties(SDL2-static PROPERTIES OUTPUT_NAME "SDL2")
   endif()
+  if(APPLE)
```

Comment:
Can you move this block to another location?
I propose just before where the first targets are created.
The current location is bad because the properties won't be applied when building no static libraries (`-DSDL_STATIC=OFF`).